### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by FK78**
Resolves #4685, #4688,  #4691, #4692, #4693, #4694, #4695, #4696, #4698

**Overall change:** _Merging dependabot PRs._

**Code changes:**

- _`package.json` and `package-lock.json` bumped._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -101,14 +101,14 @@
...
     "moment-timezone": "^0.5.27",
     "morgan": "^1.9.1",
-    "ora": "^4.0.2",
+    "ora": "^4.0.3",
     "path-to-regexp": "^3.1.0",
     "polyfill-crypto.getrandomvalues": "1.0.0",
     "ramda": "^0.26.1",
-    "react": "^16.11.0",
-    "react-dom": "^16.11.0",
+    "react": "^16.12.0",
```
Comment (chris-hinds) [inline package.json]: ld pull out react updates into a separate PR, we want to ensure that Psammead is up to date on react first before updating the version in Simorgh

File: package.json [hunk 2]
```diff
@@ -158,29 +158,29 @@
...
     "jsdom": "^15.2.1",
-    "lint-staged": "^9.4.2",
+    "lint-staged": "^9.4.3",
     "npm-run-all": "^4.1.5",
     "prettier": "^1.19.1",
     "prop-types": "^15.7.2",
     "start-server-webpack-plugin": "^2.2.5",
     "storybook-chromatic": "^3.1.0",
-    "stylelint": "^11.1.1",
+    "stylelint": "^12.0.0",
```
Comment (chris-hinds) [inline package.json:183]: This is a major change, worth checking the changelog to see if we expect any differences

**Review by chris-hinds — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (FK78) [reply package.json]: [Done](https://github.com/bbc/simorgh/pull/4712)

File: package.json (hunk 2 above)
Comment (FK78) [reply package.json:183]: From reading the [changelog](https://github.com/stylelint/stylelint/releases/tag/12.0.0) it looks like the changes do not impact us.

**Review by jroebu14 — APPROVED** [review]
looks good

**Review by mykeels — APPROVED** [review]
